packages/uikit-workshop/src/scripts/components/pl-nav/pl-nav.js

```diff
@@ -370,7 +219,22 @@ class Nav extends BaseComponent {
 
   toggleNavPanel(e) {
     const target = e.target;
+
     target.classList.toggle('pl-is-active');
+
+    // when the Nav renders as a dropdown menu, only allow one top-level menu item to be open at a time to prevent overlap issues
+    if (this.layoutMode !== 'vertical' && window.innerWidth > 670) {
+      this.topLevelTriggers = document.querySelectorAll(
+        '.pl-c-nav__link--title.pl-is-active'
+      );
+
+      this.topLevelTriggers.forEach(trigger => {
+        if (trigger !== target) {
+          trigger.classList.remove('pl-is-active');
+          trigger.nextSibling.classList.remove('pl-is-active');
+        }
+      });
```

Comment:
If you walk through all trigger, why you need to remove the `pl-is-active` class from the next sibling?

Reply:
This chunk of code was just copied over from what we were already doing in the full Nav cleanup method. 

Keep in mind that despite changing how the Nav was previously being rendered, much of the current code here hasn't been fully refactored (hence there's still some cruft to clean up like what you're pointing out here).